openmmforcefields/generators/template_generators.py

```diff
@@ -1603,6 +1603,7 @@ def __init__(self, molecules=None, cache=None, forcefield=None, model_cache_path
             self._reference_forcefield = template_generator_kwargs.get('reference_forcefield', 'openff_unconstrained-2.0.0')
             self._charge_method = template_generator_kwargs.get('charge_method', 'nn')
         else:
+            # Consider upgrading to 2.1.0, the recommended small moleucle force field for general use
             self._reference_forcefield = 'openff_unconstrained-2.0.0'
             self._charge_method = 'from-molecule'
```

Comment:
I noticed this in `EspalomaTemplateGenerator` but didn't want to make the change without better understanding what would happen

Reply:
I believe the LJ parameters are unchanged between 2.0.0 and 2.1.0, so this should have no effect. But thanks for adding the comment for us to reevaluate later!

Reply:
I think that's right - it's a Pavlovian response of mine 🙂